cookbooks/global_melt.prm

```diff
@@ -117,7 +117,7 @@ subsection Discretization
   end
 end
 
-##################### Initial conditions ########################
+##################### Initial temperature model ########################
```

Comment:
I think this old heading is more appropriate (we handle temperature and composition)